main.css

```diff
@@ -0,0 +1,115 @@
+:root {
+    --pink: #f7e1d7ff;
+    --main: rgb(255, 233, 210);
+    --green: #b0c4b1ff;
+    --pretty: rgb(121, 159, 117);
+    background-color: lightblue;
+  }
+  
+  html {
+    height: 100%;
+  }
+  
+  body {
+    height: 100%;
+    background-color: var(--main);
+  }
```

Comment:
Right now, the background color only goes to the height of your screen, but once you scroll down on the homepage the tan background goes away. There's not really a simple fix for this, I would just recommend googling what you're trying to do and the issues you're running in to.